Apr SL release notes (#5373)
dbt-labs/docs.getdbt.com: website/docs/docs/dbt-versions/release-notes/70-Apr-2024/apr-sl.md

---
title: "dbt Semantic Layer updates and fixes for April 2024"

> description: "Apr 2024: Enhanced Google Sheets integration include new features for metric labels, saved selections, and time filters. Tableau integration improvements include data source update and bug fixes."

Author: ```suggestion
description: "Apr 2024: Enhanced Google Sheets integration include new features for metric labels, saved selections, and time filters. Tableau integration improvements include data source updates."
```

sidebar_label: "Updates and fixes: dbt Semantic Layer"
sidebar_position: 06
tags: [Apr-2024]
date: 2024-04-30
---

The dbt Labs team continues to work on adding new features, fixing bugs, and increasing reliability for the [dbt Semantic Layer](/docs/use-dbt-semantic-layer/dbt-sl) and [MetricFlow](/docs/build/about-metricflow). This release note includes fixes, updates, and deprecations for in April 2024.

:::info Updated release note layout

> We've updated the layout of our release notes to provide a single view of the changes. Check out the [updated layout](/docs/dbt-versions/2024-release-notes) and let us know what you think by [opening up a GitHub issue](https://github.com/dbt-labs/docs.getdbt.com/issues)!

Reviewer: ```suggestion
We've updated the layout of our release notes to provide a single view of the changes. Check out the [updated layout](/docs/dbt-versions/2024-release-notes) and let us know of any problems by [opening up a GitHub issue](https://github.com/dbt-labs/docs.getdbt.com/issues)!
```

:::

### New features

**Google Sheets**
- **Saved selections** &mdash; You can [save your query selections](/docs/use-dbt-semantic-layer/gsheets#using-saved-selections) within the [Google Sheets application](/docs/use-dbt-semantic-layer/gsheets). They can be made private or public and refresh upon loading.
- **Display metric label** &mdash; Metrics are now displayed by their labels as `metric_name`.

### Updates

- **Metadata in metrics** &mdash; [Metrics](/docs/build/metrics-overview) now supports the [`meta` option](/reference/resource-configs/meta) under the [config](/reference/resource-properties/config) property. Previously, we only supported the now deprecated `meta` tag.

**Google Sheets**

- **Explore saved queries** &mdash; Added [support](/docs/use-dbt-semantic-layer/gsheets#using-saved-queries) to allow jumping off from or exploring MetricFlow-defined saved queries directly.

> - **Query dimensions** &mdash; Added support to query dimensions without metrics. Previously, you needed a dimension.

Author: waiting on @DevonFulcher to confirm if this made it

Author: Confirmed it made it!
